src/wp-includes/class-http.php

```diff
@@ -319,6 +327,7 @@ public function request( $url, $args = array() ) {
 		$type    = $parsed_args['method'];
 		$options = array(
 			'timeout'   => $parsed_args['timeout'],
+			'connect_timeout'   => $parsed_args['connect_timeout'],
```

Comment:
@bdanchilla , can you please update the indentation of the `$options` array definition based on https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#indentation?